### PR TITLE
Change Tag class

### PR DESCRIPTION
**Pull request by eugenetayyj**

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/CreateCommand.java [hunk 1]
```diff
@@ -35,7 +35,7 @@ public class CreateCommand extends UndoableCommand {
             + PREFIX_EMAIL + "johnd@example.com "
             + PREFIX_ADDRESS + "311, Clementi Ave 2, #02-25 "
             + PREFIX_NETWORTH + "$1423 "
-            + PREFIX_MEETING_TIME + "15-12-2022 19:00"
+            + PREFIX_MEETING_TIME + "12-15-2022 19:00 "
```
Comment (ReaganTan00) [inline src/main/java/seedu/address/logic/commands/CreateCommand.java:38]: Not sure about the change here and why it was made

File: src/main/java/seedu/address/model/tag/Tag.java [hunk 2]
```diff
@@ -29,7 +32,8 @@ public Tag(String tagName) {
      * Returns true if a given string is a valid tag name.
      */
     public static boolean isValidTagName(String test) {
-        return test.matches(VALIDATION_REGEX);
+        return test.equals(CUSTOMER_STATUS.SECURED.toString())
+                || test.equals(CUSTOMER_STATUS.POTENTIAL.toString());
```
Comment (ReaganTan00) [inline src/main/java/seedu/address/model/tag/Tag.java:36]: We can have shortcuts for this too

**Review by ReaganTan00 — APPROVED** [review]
LGTM

File: src/main/java/seedu/address/logic/commands/CreateCommand.java (hunk 1 above)
Comment (eugenetayyj) [reply src/main/java/seedu/address/logic/commands/CreateCommand.java:38]: It failed the date time format. and i introduced a space at the end cuz need to leave a space before putting the tag

**Review by ReaganTan00 — COMMENTED** [review]
LGTM

**Review by ReaganTan00 — APPROVED** [review]
LGTM

**Pull request merged**